plugins/generator-1.18.2/forge-1.18.2/procedures/entity_pathfind_to.java.ftl

```diff
@@ -0,0 +1,2 @@
+if (${input$entity} instanceof PathfinderMob _entity)

```

Comment:
Seems all Mobs have navigation, not just pathfinder ones. Is there a reason you decided to only cover PathfinderMob's? I did not check, that is why I am asking, but there is probably a reason Mob has getNavigation too

Reply:
I tried LivingEntity and that didnt't have getNavigation, seems that Mob does and extends LivingEntity, I'll change it to that.